test/e2e/versionskew/versionskew.go

```diff
@@ -39,8 +39,8 @@ import (
 )
 
 const (
-	// base is the release branch used for version skew testing.
-	base = "0.7"
+	// base is the release branch used for version skew testing. Empty if none.
+	base = "0.8"
```

Comment:
I didn't mean to disable version skew testing on "devel" - that came from cherry-picking the 0.8 commit. In "devel", we need to bump testing to testing against 0.8. Fixed.